data/helpers.d/php

```diff
@@ -2,7 +2,8 @@
 
 # Declare the actual php version to use.
 # A packager willing to use another version of php can override the variable into its _common.sh.
-YNH_PHP_VERSION=${YNH_PHP_VERSION:-$YNH_DEFAULT_PHP_VERSION}
+YNH_DEFAULT_PHP_VERSION=${YNH_DEFAULT_PHP_VERSION:-7.0}
```

Comment:
```suggestion
YNH_DEFAULT_PHP_VERSION=7.0
```
Obviously not defined since the purpose of this line is to define it.
Also, to force the definition is a good way to be sure no apps script can change that value in its _common.

Reply:
Since I'm going to remove `YNH_DEFAULT_PHP_VERSION` from the core, it seems to me that it's legit.